gradle/jacoco.gradle

```diff
@@ -60,5 +60,5 @@ afterEvaluate {
   }
 
   jacocoTestCoverageVerification.dependsOn jacocoTestReport
-  check.dependsOn jacocoTestCoverageVerification
+//  check.dependsOn jacocoTestCoverageVerification
```

Comment:
is this still needed now that jacoco verification is fixed in #357 (i just merged master into this PR)

Reply:
No, reverted.